docs/spec/api/0.1/recommendations_for_activity_streams.md

```diff
@@ -634,9 +634,9 @@ A new entity _MUST_{:.strong-term} be implemented as an _Activity_{:.term} follo
 
 An entity appearing in an _Entry Point_{:.term} stream for the first time _MUST_{:.strong-term} use _Activity_{:.term} type _Create_{:.term} or _Add_{:.term}.
 
-_Create_{:.term} _SHOULD_{:.strong-term} be used when the entity is new in the source dataset.
+_Create_{:.term} _SHOULD_{:.strong-term} be used when the entity is new in the source dataset and available for use. A provider MUST NOT use _Create_{:.term} to broadcast that an entity exists if they don't want consumer to use the entity yet; see _Add_{:.term} below.
```

Comment:
I this the key thing here "A provider MUST NOT use _Create_ to broadcast that an entity exists unless it can be dereferenced at the entity URI. A _Create_ activity indicates that the entity is new and available for use by consumers, see also _Add_ below."

I like to avoid normative languages tied to wishes and intentions -- prefer tied to technical requirements (available for dereference)

Reply:
Sounds good. And in theory they won't publish the URI until they are comfortable with someone dereferencing it.

Reply:
Incorporated feedback here: https://github.com/LD4/entity_metadata_management/pull/76/commits/2c930d39676904b375d925331b59034b680741b4